lib/literature/components/table_component.ex

```diff
@@ -176,6 +174,37 @@ defmodule Literature.TableComponent do
     """
   end
 
+  defp create_icon(assigns) do
+    ~H"""
+    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" fill="currentColor" class="w-5 h-5">
+      <path
+        fill-rule="evenodd"
+        d="M12 2.25c-5.385 0-9.75 4.365-9.75 9.75s4.365 9.75 9.75 9.75 9.75-4.365 9.75-9.75S17.385 2.25 12 2.25zM12.75 9a.75.75 0 00-1.5 0v2.25H9a.75.75 0 000 1.5h2.25V15a.75.75 0 001.5 0v-2.25H15a.75.75 0 000-1.5h-2.25V9z"
+        clip-rule="evenodd"
+      />
+    </svg>
+    """
+  end
```

Comment:
Have you consider creating a module that renders a component that will render the svg icon that you need?
Something like
"<Icon.render name="create-icon" />"